addons/l10n_vn/migrations/2.0.4/post-migration.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, SUPERUSER_ID
+from odoo.osv import expression
+
+FIXED_ACCOUNTS_TYPE = {
+    'liability_current': ['3387'],
+}
+
+
+def _fix_accounts_type(env):
+    for correct_account_type, accounts_code in FIXED_ACCOUNTS_TYPE.items():
+        domains_per_company = []
+        for company in env['res.company'].with_context(active_test=False).search([('chart_template', '=', 'vn')]):
+            doamin = expression.AND([
+                [('company_id', '=', company.id), ('account_type', '!=', correct_account_type)],
+                expression.OR([
+                    [('code', 'like', f'{code}%')] for code in accounts_code
+                ])
```

Comment:
I don't think we will add more accounts to correct in this one migration script, so you can simplify it by removing the FIXED_ACCOUNTS_TYPE dict and this or here.

Also, wouldn't it be easier to read if you get the ids of all VN companies and write the domain with a company_id in ids 🤔